rd-net/Lifetimes/Util/LocalStopwatch.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using JetBrains.Diagnostics;
+
+namespace JetBrains.Util
+{
+  public readonly struct LocalStopwatch
+  {
+    #region Statics
+
+    private const long TicksPerMillisecond = 10000;
+    private const long TicksPerSecond = TicksPerMillisecond * 1000;

```

Comment:
Please add a comment that these values are synchronized with the internal values of `Stopwatch` from the BCL.